javascript/spliffy/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "@srfnstack/spliffy": "^0.7.0"
```

Comment:
please use `~` to stick to the same minor version

```suggestion
    "@srfnstack/spliffy": "~0.7.0"
```

Reply:
Updated. Thanks for checking.